modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+
+Outbound - Ports::
+The sum of outbound requests, grouped by remote port in automatic intervals.
+
+// ports?
+
+Outbound - Operations::
+The sum of outbound requests, grouped by operation in automatic intervals.
+
+Outbound - Protocols::
+The sum of outbound requests, grouped by protocol in automatic intervals.
+
+Outbound - Endpoints::
+The sum of outbound requests, grouped by remote hosts (endpoints) in automatic intervals.
+
+[[flows-charts]]
+== Flows
+
+The following charts appear on the *Flows* dashboard:
+
+Inbound Response Time by Flow::
+The average inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The sum of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The sum of outbound requests, grouped by flow.
+
+Failures by Flow::
+The sum of failed inbound requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The sum of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Inbound Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The sum of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The sum of outbound requests, grouped by flow.
+
+Failures by Flow::
+The sum of inbound failed requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The sum of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+[[connectors-charts]]
+== Connectors
+
+If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
+
+The following charts appear on the *Connectors* dashboard:
+
+Total Connector Requests::
+The sum of outbound requests grouped by connector type in automatic intervals.
+
+Response Time by Connector::
+The average outbound response time grouped by connector type in automatic intervals.
+
+Salesforce Requests::
+The average number of Salesforce connector requests, grouped by operation in automatic intervals.
+
+Salesforce Response Time::
+The average response time of Saleforce connector requests, grouped in automatic intervals.
+
+Salesforce Failures::
+The average number of Salesforce connector requests failures, grouped by operation in automatic intervals.
+
+Database Requests::
+The average number of Database connector requests, grouped by operation in automatic intervals.
```

Comment:
ditto: do we mean "Anypoint Connector for Database (Database Connector)"?

Reply:
```suggestion
The average number of Anypoint Connector for Database (Database Connector) requests, grouped by operation in automatic intervals. For more information about which operations appear, see the section about xref:monitoring::app-dashboards.adoc#endpoint-operation-metrics[endpoint and operation data collection]
```